### DISCUSSION TITLE
Monitor the Door Micro Switch

### DISCUSSION THREAD

**Discussion by Sideline-Review** [greghesp/ha-bambulab#165] [Ideas]
It would be great to be able to monitor the door micro switch. I would like to have my Bento Box air scrubber to turn on during a print when the door is closed. I can't seem to find anywhere that shows the status of that switch. I think originally it was designed to pause the print when the door was opened.

**Comment by AdrianGarside** [comment]
Unfortunately the door switch isn't exposed in the mqtt payload so we are unable to add this currently.

**Comment by greghesp** [comment]
As Adrian says, unfortunately not exposed yet
https://github.com/greghesp/ha-bambulab/issues/127
